packages/qdrant-loader/src/qdrant_loader/connectors/jira/connector.py

```
@@ -451,36 +451,46 @@ async def fetch_by_id(self, entity_id: str) -> Document | None:
         """Fetch a single Jira issue by key or numeric ID."""
         try:
             raw_issue = await self._make_request("GET", f"issue/{entity_id}")
+            issue = self._parse_issue(raw_issue, self.config.extra_fields)
+            documents = await self._issues_to_documents(
+                [issue], include_attachments=False
+            )
         except Exception as exc:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)